Cookbook/Technical-Documents/ToolsAndServices.md

```diff
@@ -7,17 +7,18 @@
 | 1Password | Access shared passwords used by various services in the team | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) Specify in the ticket that access to iOS team vault is needed |
 | JIRA | Tickets of tasks and issues on your sprint | You should get an invitation email to join the BabylonPartners team after the [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) has been resolved |
-| Zeplin | Screen Designs | Ask the support engineer in the iOS team channel on Slack for access to the project(s) |
-| GitHub | Repos for the app, the bots, the playbook... | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/248) to get added to the org. Make sure your Babylon email is added to your account. |
+| Zeplin | Screen Designs | Ask the support engineer in the iOS team channel on Slack for access to the project(s). Depending on your tribe you might need to use Figma |
+| [Figma](https://www.figma.com/) | Screen Designs | Log in with Google as this should redirect to the Babylon Okta sign in. Depending on your tribe you might need to use Zeplin |
+| GitHub | Repos for the app, the bots, the playbook... | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/248) to get added to the org. Make sure you add your GitHub username in the support ticket, and that your Babylon email is added to your account |
 | [Lokalise](https://lokalise.co/projects) | Manage translations used by the app | Use iOS team vault on 1Password to gain access |
 | Confluence Spaces (iOS, your Tribe, ...) | Documentation for various processes | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) – You can also ask your PM or Tribe Lead which Spaces you need to be given access to. (Since we are moving away from Confluence this might not be necessary) |
 | Google Team Drives | Access shared documents for which you need multiple people to contribute | Login using your Microsoft account which is used for email – You can also ask your PM / Tribe Lead which Drives you need to be given access to. You should be given access to at least `iOS` and `Consumer Apps Tribe`|
 | [zoom.us](https://zoom.us) | Video Conferences | Create account with BabylonHealth email-Id. You will have access to basic plan (you can join invites without an account but will need one to create conferences)|
 | VPN | Remote Access to our intranet and internal domains | [IT Support ticket](https://supportbybabylon.atlassian.net/servicedesk/customer/portal/5/group/12/create/43) |
 | [CircleCI](https://circleci.com/gh/babylonhealth) | See CI jobs and logs | Use your GitHub login |
-| [Pull Panda](https://pullreminders.com) | Stats on PRs and reminders for when you've been asked for a review | Use your GitHub login |
```

Comment:
We should keep Pull Panda, we still have stats there

Reply:
I can revert it, but is that going to be useful if new hires can't use it? I can't see those stats for example as it doesn't allow new sign ups 🙁

Reply:
Oh you can't sign up anymore? That's new

Reply:
Yeah, forgot to add it in the PR description (will add it now), but when I try to sign in, it shows this banner that links to this [article](https://pullpanda.com/github) about the acquisition by GitHub.

<img width="456" alt="Screenshot 2020-03-03 at 09 11 23" src="https://user-images.githubusercontent.com/3620405/75760392-1e372d00-5d2f-11ea-8098-c49c374d0e9e.png">

Reply:
:sad-panda: it should be removed then